@nedbat here's [an example](https://travis-ci.org/edx/cookiecutter-django-ida/builds/98171115) of the attribute error.

```
pylint --rcfile=pylintrc repo_name *.py
Traceback (most recent call last):
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/bin/pylint", line 11, in <module>
    sys.exit(run_pylint())
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/pylint/__init__.py", line 23, in run_pylint
    Run(sys.argv[1:])
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/pylint/lint.py", line 1332, in __init__
    linter.check(args)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/pylint/lint.py", line 747, in check
    self._do_check(files_or_modules)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/pylint/lint.py", line 869, in _do_check
    self.check_astroid_module(ast_node, walker, rawcheckers, tokencheckers)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/pylint/lint.py", line 946, in check_astroid_module
    walker.walk(ast_node)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/pylint/utils.py", line 874, in walk
    self.walk(child)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/pylint/utils.py", line 871, in walk
    cb(astroid)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/pylint/checkers/typecheck.py", line 293, in visit_assign
    function_node = safe_infer(node.value.func)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/pylint/checkers/utils.py", line 92, in safe_infer
    value = next(inferit)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/bases.py", line 327, in wrapped
    for res in _func(node, context, **kwargs):
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/bases.py", line 351, in wrapper
    for node in func(*args, **kwargs):
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/inference.py", line 241, in infer_getattr
    for owner in self.expr.infer(context):
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/bases.py", line 86, in cache_generator
    for result in generator:
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/bases.py", line 327, in wrapped
    for res in _func(node, context, **kwargs):
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/bases.py", line 124, in _infer_stmts
    for infered in stmt.infer(context):
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/bases.py", line 86, in cache_generator
    for result in generator:
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/bases.py", line 327, in wrapped
    for res in _func(node, context, **kwargs):
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/bases.py", line 124, in _infer_stmts
    for infered in stmt.infer(context):
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/bases.py", line 86, in cache_generator
    for result in generator:
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/bases.py", line 327, in wrapped
    for res in _func(node, context, **kwargs):
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/inference.py", line 229, in infer_from
    module = self.do_import_module()
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/mixins.py", line 107, in do_import_module
    return mymodule.import_module(modname, level=level)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/scoped_nodes.py", line 431, in import_module
    return MANAGER.ast_from_module_name(modname)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/manager.py", line 160, in ast_from_module_name
    return self.ast_from_file(filepath, modname, fallback=False)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/manager.py", line 112, in ast_from_file
    return AstroidBuilder(self).file_build(filepath, modname)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/builder.py", line 134, in file_build
    module = self._data_build(data, modname, path)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/builder.py", line 177, in _data_build
    module = rebuilder.visit_module(node, modname, node_file, package)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/rebuilder.py", line 150, in visit_module
    return self._transform(newnode)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/astroid/manager.py", line 337, in transform
    ret = transform_func(node)
  File "/home/travis/build/edx/cookiecutter-django-ida/repo_name/.venv/local/lib/python2.7/site-packages/pylint_django/transforms/__init__.py", line 29, in set_fake_locals
    module._locals[class_name] = fake._locals[class_name]  # pylint: disable=protected-access
AttributeError: 'Module' object has no attribute '_locals'
```

Looks like `pylint_django` might be at fault.

I'm not sure what you mean by "how you referenced edx-lint before." It was included in the project's requirements file without being pinned to a version.

--- a/{{cookiecutter.repo_name}}/requirements/test.txt
+++ b/{{cookiecutter.repo_name}}/requirements/test.txt
@@ -7,7 +7,5 @@ django-nose
 mock
 nose-ignore-docstring
 pep8
-pylint < 1.5
-astroid < 1.4
 
 edx-lint

--- a/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/apps/core/models.py
+++ b/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/apps/core/models.py
@@ -25,3 +25,6 @@ class User(AbstractUser):
 
     def get_full_name(self):
         return self.full_name or super(User, self).get_full_name()
+
+    def __unicode__(self):
+        return unicode(self.get_full_name())

--- a/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/apps/core/tests/test_models.py
+++ b/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/apps/core/tests/test_models.py
@@ -38,3 +38,9 @@ class UserTests(TestCase):
 
         user = G(User, full_name=full_name, first_name=first_name, last_name=last_name)
         self.assertEquals(user.get_full_name(), full_name)
+
+    def test_unicode(self):
+        """Verify that the model's unicode method returns the user's full name."""
+        full_name = "Bob"
+        user = G(User, full_name=full_name)
+        self.assertEquals(unicode(user), full_name)

--- a/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/urls.py
+++ b/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/urls.py
@@ -42,5 +42,5 @@ urlpatterns = [
 ]
 
 if settings.DEBUG and os.environ.get('ENABLE_DJANGO_TOOLBAR', False):  # pragma: no cover
-    import debug_toolbar  # pylint: disable=import-error
+    import debug_toolbar  # pylint: disable=import-error,wrong-import-position,wrong-import-order
     urlpatterns.append(url(r'^__debug__/', include(debug_toolbar.urls)))

